contracts/Liquidation.sol

```diff
@@ -27,8 +27,8 @@ contract Liquidation is ILiquidation, Ownable {
     uint256 public override minimumLeftoverGasCostMultiplier = 10;
     IPricing public pricing;
     ITracerPerpetualSwaps public tracer;
-    address public insuranceContract;
-    address public fastGasOracle;
+    address public immutable insuranceContract;
+    address public immutable fastGasOracle;
```

Comment:
This is actually my bad from the PR that I made for adding a fast gas oracle to `Liquidation`, but you should be able to change the address of this. Could you add a setter to it please? (and consequently remove the immutable keyword)

Reply:
@raymogg @BenjaminPatch Do we creep scope or would we prefer another PR? Particularly worried about correspondence with Code Area issues.

Reply:
That is actually true; probably good to leave it to another PR.